state/execution_test.go

```diff
@@ -582,6 +584,11 @@ func TestEndBlockValidatorUpdatesResultingInEmptySet(t *testing.T) {
 		{PubKey: vp, Power: 0},
 	}
 
+	// dYdX fork: Apply block should lock/unlock the mempool.
```

Comment:
for my own understanding, could you point me to where Lock/Unlock/FlushAppConn were added?

Reply:
Yes it's [this](https://github.com/dydxprotocol/cometbft/commit/4eea1903d5506da5eebf8f36045957c19da2d182) commit.